client/src/pages/Login/Login.tsx

```diff
@@ -36,6 +36,18 @@ export default function Login(): JSX.Element {
     });
   };
 
+  const handleDemoLogin = () => {
+    const data = {
+      email: 'doglover@gmail.com',
+      password: 'dogLover',
+    };
+    demoUserLogin(data.email, data.password).then((data) => {
+      if (data.success) {
+        updateLoginContext(data.success);
+      }
```

Comment:
If there's an error logging in, can we please handle that use case here and accordingly update the snack bar?